src/NServiceBus.Transport.SQS.Tests/MessageDispatcherTests.cs

```diff
@@ -926,8 +926,42 @@ public async Task Should_handle_empty_body_gracefully(bool wrapMessage)
             Assert.That(request.MessageBody, Is.Not.Null.Or.Empty);
         }
 
-        interface IEvent { }
+        [Test]
+        public async Task Does_bump_to_S3_when_payload_padding_is_set()
```

Comment:
I haven't checked all the tests for a consistent verbiage, but on a superficial level, it looks like we use the term "upload" instead of bump.